deploy/cloudformation/static-host-pipeline.yml

```diff
@@ -136,22 +132,22 @@ Resources:
             install:
               commands:
                 - echo Install started on `date`
-                - chmod -R 755 build/*
-                - ./build/install.sh
+                - chmod -R 755 scripts/build/*
```

Comment:
@jonhartzler The mellon-website repo still has scripts/build, but we discussed scripts/codebuild. Is this going to change?

Reply:
This will be changed with https://github.com/ndlib/mellon-website/pull/3